src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -8407,7 +8407,10 @@ public void ParamsArray_NotArray()
             CreateCompilation(source).VerifyDiagnostics(
                 // (1,12): error CS0225: The params parameter must be a single dimensional array
                 // var lam = (params int x) => x;
-                Diagnostic(ErrorCode.ERR_ParamsMustBeArray, "params").WithLocation(1, 12));
+                Diagnostic(ErrorCode.ERR_ParamsMustBeArray, "params").WithLocation(1, 12),
+                // (1,23): warning CS9100: Parameter 1 has params modifier in lambda but not in target delegate type.
```

Comment:
>// (1,23): warning CS9100: Parameter 1 has params modifier in lambda but not in target delegate type. [](http://example.com/codeflow?start=16&length=101)

Perhaps we should suppress this warning for unconventional `params` applications.

Reply:
I assume we can get in this situation with method groups too, but didn't see a test with a similar change in behavior for them.